docs/source/configuration/telemetry/instrumentation/conditions.mdx

```diff
@@ -6,7 +6,7 @@ description: Set conditions for when events or instruments are triggered in the
 
 <PremiumFeature linkWithAnchor="https://www.apollographql.com/pricing#observability"/>
 
-You can set conditions for when an [instrument](./instruments) should be mutated or an [event](./events) should be triggered.
+You can set conditions for when an [instrument](./instruments) should be mutated <!--or an [event](./events) -->should be triggered.
```

Comment:
You added HTML comments. Is it intentional?

Reply:
Yes, that's how it's done in other document so I just reproduced the same :p and it works